opticsApp/iocsh/flexCombinedMotion.iocsh

```diff
@@ -0,0 +1,37 @@
+# ### flexCombinedMotion.iocsh ###
+
+#- ###################################################
+#- P                - IOC Prefix
+#- C                - Cap sensor name (cap1)
+#- V                - Cap sensor voltage PV (PREFIX:LJT8:1:Ai0)
+#- UMV              - Cap sensor microns per volt (32.5)
+#- OFF              - Optional: Cap sensor voltage offset
+#-                    Default: 0.0
+#-
+#- PREC             - Optional: Cap Sensor precision
+#-                    Default: 5
+#-
+#- EGU              - Optional: Cap Sensor and cap sensor motor EGU
+#-                    Default: um
+#-
+#- M                - Cap sensor motor name (m1)
+#- DESC             - Cap sensor motor description ("flexure motor")
```

Comment:
Does DESC need to be a required value?  Since it is easily editable in the motor3x screen and then saved by autosave, I feel like you could have a default value here and make DESC optional to save some screen real estate on the iocshLoad line.

Reply:
Technically it isn't required; the support will work with a default description.  However, the databases in motor have always required descriptions and beamlines always give motors meaningful names:

https://github.com/epics-modules/motor/blob/34474ed958838ea5083f598fa4bd9f1ca7e1821c/motorApp/Db/asyn_motor.db#L7

I'll make it optional.  Long lines are annoying.